plugin/rbx_dom_lua/EncodedValue.lua

```diff
@@ -11,13 +11,44 @@ local function unpackDecoder(f)
 end
 
 local function serializeFloat(value)
```

Comment:
Does this need to be contributed upstream? I'm not clear on whether the copy of rbx_dom_lua in rojo is fully synced with upstream.